core/src/test/scala/org/apache/spark/scheduler/PoolSuite.scala

```diff
@@ -201,6 +202,49 @@ class PoolSuite extends SparkFunSuite with LocalSparkContext {
     verifyPool(rootPool, "pool_with_surrounded_whitespace", 3, 2, FAIR)
   }
 
+  test("SPARK-19466: Fair Scheduler should build fair scheduler when " +
```

Comment:
Could you actually move these test changes to a separate PR (and no need to include the JIRA name since this isn't fixing a bug)?  Awesome to add test coverage but it seems separate from the logging improvement.